src/products/views/ProductList/ProductList.tsx

```diff
@@ -283,20 +309,24 @@ export const ProductList: React.FC<ProductListProps> = ({ params }) => {
 
   const filterOpts = getFilterOpts(
     params,
-    initialFilterData?.attributes?.edges?.map(edge => edge.node) || [],
+    initialFilterAttributes?.attributes?.edges?.map(edge => edge.node) || [],
```

Comment:
Considering how many times we do this `edges?.map(edge => edge.node)` in the app, maybe we can add a util function?